src/charm.py

```diff
@@ -518,6 +519,19 @@ def _on_config_changed_with_warning(self, event: Any) -> None:
             )
             self.unit.status = status.from_name(status.name, warning)
 
+    def _on_start(self, event: Any) -> None:
```

Comment:
It'd probably be better to use an underscore instead of `event` and don't document the arg in the docstring

Reply:
Good point, updated.